Makefile

```diff
@@ -3,11 +3,11 @@ docker-build:
 	# Build dev docker images.
 	# Having issues? try: docker volume prune
 	# Still having issues? try: docker system prune
-	docker-compose build
+	docker compose build
 
 docker-run:
 	# Run docker in dev mode.
-	docker-compose up
+	docker compose up
 
 docker-db:
```

Comment:
There's an old `docker-compose` in this command

Reply:
In that case - updating all the other references too.